pkg/util/api/api.go

```diff
@@ -16,6 +16,10 @@ limitations under the License.
 
 package api
```

Comment:
I'm not convinced about putting this function here. But we can revisit once we have the jobs library completed, which will probably use it.